packages/sources/svelte/src/components/actions/VtmnDropdown/VtmnDropdown.svelte

```diff
@@ -0,0 +1,147 @@
+<script>
+  import { cn } from '../../../utils/classnames';
+  import { createEventDispatcher } from 'svelte';
+  import { clickOutside } from '../../../utils/clickOutside';
+
+  /** @restProps */
+
+  /**
+   * Custom classes to apply to the component.
+   * @type {array} options
+   */
+  export let options;
+
+  /**
+   * The main label. If not set the label is not displayed
+   *
+   * @type {string} [label]
+   */
+  export let label;
+
+  /**
+   * Default dromdown label
+   *
+   * @type {string} defaultOption
+   */
+  export let defaultOption;
+
+  /**
+   * Custom classes to apply to the component.
+   * @type {string} className
+   */
+  let className = '';
+
+  export { className as class };
+
+  /**
+   * Menu item divider.
+   *
+   * @type {boolean} [divider]
+   * @defaultValue false
+   */
+  export let divider = false;
+
+  /**
+   * Set disabled state of list item.
+   *
+   * @type {boolean} [disabled]
+   * @defaultValue false
+   */
+  export let disabled = false;
+
+  /**
+   * Icon for each menu item.
+   *
+   * @type {string} [icon]
+   */
+  export let icon;
+
+  /**
+   * Max height of menu, this enable scroll.
+   * @type {number} [menuMaxHeight]
+   */
+  export let menuMaxHeight = null;
+
+  $: componentClass = cn('vtmn-dropdown', className);
+
+  const dispatch = createEventDispatcher();
+
+  let selectedOptions = [];
+
+  /**
+   * Add or remove elements in options array.
+   *
+   * @param {string} value Selected value
+   */
+  const onSelectItem = (value) => {
+    if (selectedOptions.includes(value))
+      selectedOptions = selectedOptions.filter((option) => option !== value);
+    else selectedOptions = [...selectedOptions, value];
+
+    dispatch('item-selected', {
+      selectedOptions,
+    });
+
+    closeMenu();
+  };
+
+  let details;
+
+  /**
+   * Close dropdown menu.
+   */
+  function closeMenu() {
+    details.open = false;
+  }
+</script>
+
+<div
```

Comment:
What happening when you open a dropdown displayed on the bottom of the page. If the bottom are greater than the `document.body.scrollHeight` ?

![image](https://user-images.githubusercontent.com/2856778/159010148-44f491bd-bd85-4d83-a251-e8b0c8e8ef14.png)

@lauthieb is the use case is correct ? 
- When the dropdown is greater than the pageHeight, it should be displayed above the input
- When the dropdown is grater than the innerHeight but under the pageHeight, it should be displayed under the input (like dkt website)

Reply:
@N0t-Simon what's your opinion please?

Reply:
@lauthieb all right, I await yout reply for finish the component

Reply:
hey @jb-thery, what you think about that? 

https://user-images.githubusercontent.com/50582328/159956259-418a27ff-c63f-47c8-a2c4-7a296310392c.mov

Reply:
Yes it's a nice option

Reply:
@jb-thery for the moment I convert to draft. Feel free to make this PR "ready for review" again when you will have made the implementation proposed by @N0t-Simon.